feather/plugin-host/src/host_calls/component.rs

```diff
@@ -76,6 +87,7 @@ pub fn entity_set_component(
     component: u32,
     bytes_ptr: PluginPtr<u8>,
     bytes_len: u32,
+    variant: u8,
```

Comment:
Might be better to expose some host calls for event tracker instead. Such that it is possible to model this as two host calls. Ie adding a component and adding the component to the event tracker.